.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: true
```

Comment:
It would be much better to avoid slowing down *all* the builds but to instead have just one builder that needs `sudo: true`. This can be done by using a build matrix to include an extra builder that does the needed things.